packages/shop/src/pages/special-offer/index.page.tsx

```diff
@@ -0,0 +1,209 @@
+import { GetServerSideProps, NextPage } from 'next';
```

Comment:
can we add this to storybook ?

Reply:
Unfortunately, it's tricky to create a storybook for the server rendered pages with our current setup.

Reply:
Unfortunately, it's tricky to create a storybook for the server-rendered pages (especially with server-side data-fetching)

Reply:
You can see preview with this URL. https://rmp-demo-shop-test-4uwn2cbqz-moloco.vercel.app/special-offer?requestId=3aC2gy854H18_KE2UfB1k